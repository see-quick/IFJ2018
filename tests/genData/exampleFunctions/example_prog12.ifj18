# Draw primary numbers from interval
def prime(a)

  divider = a - 1
  return = 0

  while divider > 1 do

      if ((a / divider) * divider) == a then
          divider = 667   #false
      else
        divider = divider - 1
      end
  end
  return = divider
end


flag = 666
print "Enter two positive integers: \n"

n1 = inputi
print "\n"
n2 = inputi
print "\n"

print "Prime numbers between ", n1 ," and ", n2 , " are "

i = n1 + 1

while i < n2 do

    flag = prime(i)

    if flag != 667 then
        print i, "\n"
    else
      i = i + 1
    end
end
