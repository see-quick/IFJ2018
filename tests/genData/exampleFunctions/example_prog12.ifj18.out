Enter two positive integers: 
4
6
Prime numbers between 4 and 6 are
5

